src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.ts

```diff
@@ -127,8 +127,12 @@ export class TokenDetailsStage {
   }
 
   private addDefaultValuesToRegistrationData(wizardType: WizardType) {
-    if (wizardType === WizardType.createPartneredDeal) {
+    if (wizardType !== WizardType.createPartneredDeal) return;
```

Comment:
@dkent600 
For the "Make an offer", we should also add default tokens in the Token details stage (because we do so for the create a partnered deal too).

SOT: Make an offer 
> Just like Partnered Deal Registration Wizard except:

https://www.notion.so/primedao/Registration-Wizard-74901348d4a443a6891ad16fa1c0b42d#d1d7b1ebc7264bdc9e2b4d05541efa5e

Currently, we don't.
![image](https://user-images.githubusercontent.com/30693990/157850851-8e094c21-4f6a-48c9-9bfd-94a7a848939e.png)

The change would be just to add `WizardType.makeAnOffer` to the if check as well

Reply:
> @dkent600
> 
> **Q: Do the following change here, or in another ticket?** (The change would be just to add `WizardType.makeAnOffer` to the if check as well)
> 
> For the "Make an offer", we should also add default tokens in the Token details stage (because we do so for the create a partnered deal too).
> 
> SOT: Make an offer
> 
> > Just like Partnered Deal Registration Wizard except:
> 
> https://www.notion.so/primedao/Registration-Wizard-74901348d4a443a6891ad16fa1c0b42d#d1d7b1ebc7264bdc9e2b4d05541efa5e
> 
> Currently, we don't. ![image](https://user-images.githubusercontent.com/30693990/157850851-8e094c21-4f6a-48c9-9bfd-94a7a848939e.png)

separate ticket please

Reply:
@dkent600 

> separate ticket

> changes requested

To be clear, I did not add the "make an offer" part to this ticket.
Was the change request about that?

Reply:
got it